packages/nextjs/app/builders/0xd4A9f1f16c971f68FeF19882CBC019E616A85095/page.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+const ProfilePage: React.FC = () => {
+  return (
+    <div className="flex justify-center items-center h-screen">
+      <div className="w-96 bg-white shadow-md rounded-lg">
+        <div className="p-6">
+          <div className="text-center">
+            <div className="flex items-center justify-center">
+              <img
```

Comment:
Could you please add  `{/* eslint-disable-next-line @next/next/no-img-element */}` above this line, because if not the lint rules are failing. 

We could have used Next Image directly but then we need to update `next.config` with remote URL pattern which is not that worth it so lets just ignore it 🙌